netcore/Makefile

```diff
@@ -10,6 +10,9 @@ $(NETCORESDK_FILE):
 build-sample:
 	dotnet build HelloWorld
 
+run-sample:
+	COMPlus_DebugWriteToStdErr=1 ./dotnet --fx-version "$(VERSION)" sample/HelloWorld/bin/Debug/netcoreapp3.0/HelloWorld.dll
```

Comment:
build-sample & run-sample can be replace with `./dotnet run --fx-version "$(VERSION)" --project sample/HelloWorld`

Reply:
Didn't work for me without #13656. I agree it should be done eventually.